LEAF_Request_Portal/report.php

```diff
@@ -58,9 +58,16 @@
 // HQ logo
 $main->assign('logo', '<img src="images/VA_icon_small.png" style="width: 80px" alt="VA logo" />');
 
-function customTemplate($tpl)
+function customTemplate($tpl, $config)
```

Comment:
Since all you're using from the config is portalPath, it would be better to just pass in portalPath, instead of taking the whole config.

Reply:
done